open_lm/params.py

```diff
@@ -237,7 +237,7 @@ def check_args(args):
             if args.remote_sync_protocol != "s3":
                 raise ValueError("Sync protocol not supported when using resume latest.")
 
-    if args.lr_scheduler != "cosine":
+    if args.lr_scheduler != "cosine" or args.lr_scheduler != "const" or args.lr_scheduler != "const-cooldown":
```

Comment:
nit: Can you replace with the below for readability?
```if args.lr_scheduler not in {"cosine", "const", "const-cooldown"}:```

Reply:
actually the current version also has a bug which causes the unit tests to fail. more generally, maybe we can move the lr_scheduler components out of this PR?

Reply:
i added the line as you wrote